city_scrapers/spiders/pitt_allegheny_league.py

```diff
@@ -0,0 +1,133 @@
+# -*- coding: utf-8 -*-
+import os
+import unicodedata
+from datetime import datetime
+
+from city_scrapers.constants import BOARD, COMMITTEE, NOT_CLASSIFIED
+from city_scrapers.spider import Spider
+
+
+class PittAlleghenyLeagueSpider(Spider):
+    name = 'pitt_allegheny_league'
+    agency_name = 'Allegheny League of Municipalities'
+    timezone = 'America/Chicago'
+    allowed_domains = ['alleghenyleague.org']
+    start_urls = ['http://alleghenyleague.org/']
```

Comment:
It looks like the full calendar of events is on this page http://alleghenyleague.org/calendar-of-events/ so we should probably scrape from there instead of the home page so that we can get more than the first 3 items. We could also go one month into the future, and you can see [`chi_low_income_housing_trust_fund`](https://github.com/City-Bureau/city-scrapers/blob/master/city_scrapers/spiders/chi_low_income_housing_trust_fund.py) for an example of this

Reply:
Thanks for the link: [`chi_low_income_housing_trust_fund`](https://github.com/City-Bureau/city-scrapers/blob/master/city_scrapers/spiders/chi_low_income_housing_trust_fund.py).  [The page](http://alleghenyleague.org/calendar-of-events/) I am scraping has a click-button to go to the next month but it looks like a javascript button without a url-link. So the url would be the same for both the current and next month. Is it still possible to scrape info from  the next month?
 https://github.com/City-Bureau/city-scrapers/blob/be48a646b75891c0e0c4f4e9ff605bf92353fd21/city_scrapers/spiders/chi_low_income_housing_trust_fund.py#L41-L47
This one looks like it is looking for a href. In my case, I think ``the action to click`` is required. Any suggestion?? I think I can invoke `onclick` in `Selenium`.

Reply:
It looks like you should be able to submit a POST request to http://alleghenyleague.org/wp-admin/admin-ajax.php. If you send these parameters it should return JSON with HTML of the table in the `data` property:

```
action: simcal_default_calendar_draw_grid
id: 417
month: (month integer)
year: (year integer)
```

Selenium is a bit of a heavy dependency, and submitting a POST request should be able to get you the same information

Reply:
@pjsier  Btw, I was trying to find this link: http://alleghenyleague.org/wp-admin/admin-ajax.php at http://alleghenyleague.org/calendar-of-events/ using `Inspect` but could not find it under Network tag. Am I looking the wrong page or wrong location? Thanks.